src/ParticleDistributions/ParticleDistributions.jl

```diff
@@ -473,38 +474,43 @@ function moment_source_helper(
     dist::ExponentialPrimitiveParticleDistribution{FT},
     p1::FT,
     p2::FT,
-    x_threshold::FT;
-    n_bins_per_log_unit = 15,
-) where {FT <: Real}
+    x_threshold::FT,
+    ::Val{n_bins} = Val(100),
```

Comment:
I remember we changed this from n_bins to n_bins_per_log_unit because in some of the examples where range of scales is larger the results were not reasonable/satisfactory (because of inaccurate numerical integrations). Would be possible to keep n_bins_per_log_unit?

Reply:
Unfortunately not, because to create an SVector the size must use something in type space and cannot use something that is computed within the body